docs/luigi-core-api.md

```diff
@@ -228,6 +228,20 @@ Luigi.reset();
 
 -   **since**: 1.14.0
 
+#### clearNavigationCache
+
+Delete cache. Clear titleResolver cache of nested navigation nodes.
```

Comment:
also (and more important) the node cache is cleared, i.e. asynchronously fetched children will be fetched again.
But i would phrase it more generic, since there might  be additional caches in the future that might also be cleared by this function... so sth like:
"Clear navigation node related caches."

Reply:
Updated the description. Thanks for your suggestion.